charts/frontend/templates/deployment.yaml

```diff
@@ -20,6 +20,8 @@ spec:
           ports:
             - name: http
               containerPort: {{ .Values.appPort }}
+          command: ["bundle"]
+          args: ["exec", "rails", "server"]
```

Comment:
Should we not specify the port here to match the `containerPort` above?

```
["exec", "rails", "server", "-p", "{{ .Values.appPort }}"]
```

or such?

Reply:
Good point but we're already setting it (perhaps in a slightly less clear way) by the `PORT` env var below. In principle we could remove `PORT` and use `-p`, but in practice that'd mean we'd have to change the Dockerfile in a way that's not compatible with its existing usage (in alphagov/publishing-e2e-tests).

Reply:
Ah, forgot about that env var - no need to change that to an arg 👍

Reply:
I think the preferred way to set the port would be to set the environment variable (https://github.com/alphagov/frontend/blob/main/config/puma.rb#L18) which is read by the config file.